gateway/1.6/modules/ROOT/pages/_partials/task-shared-storage-config.adoc

```diff
@@ -107,10 +112,46 @@ spec:
 ----
 //end::sample-config-all[]
 
+//tag::sentinel-intro[]
+[[redis-sentinel]]
+== Configure Redis Sentinel
+
+Redis Sentinel provides high availability for Redis shared storage. For more information about Redis Sentinel, see https://redis.io/docs/management/sentinel/[Redi Sentinel^].
+
+Configure Redis Sentinel using the steps described in the previous sections.
+
+Sample configuration for adding Redis Sentinel:
+//end::sentinel-intro[]
+
+//tag::sentinel-sample-config[]
+[source, yaml]
+----
+apiVersion: gateway.mulesoft.com/v1alpha1
+kind: Configuration
+metadata:
+  name: shared-storage
+  namespace: test-ns
+spec:
+  sharedStorage:
+    redis:
+      username: "user"
+      password: "pass"
+      sentinel:
+        addresses:
+          - internal.redis.com:6379
+        masterName: "name"
```

Comment:
It looks like the PR was flagged because of the use of "master". Is this something that we can change? Or is it a Redis param? We won't be able to merge the 1.6 branch into latest.